extensions/viem/src/tevmViemExtension.js

```diff
@@ -271,6 +271,36 @@ export const tevmViemExtension = () => {
 			)
 		}
 
+		/**
+		 * @type {import('@tevm/api').DumpStateHandler}
+		 */
+		const dumpState = async () => {
+			return /** @type {any} */ (
+				formatResult(
+					await request({
+						method: 'tevm_dump_state',
+						jsonrpc: '2.0',
+						params: undefined,
+					}),
+				)
+			)
+		}
+
+		/**
+		 * @type {import('@tevm/api').LoadStateHandler}
+		 */
+		const loadState = async (params) => {
+			return /** @type {any} */ (
+				formatResult(
+					await request({
+						method: 'tevm_load_state',
+						jsonrpc: '2.0',
+						params,
```

Comment:
```suggestion
						params,
```
Since JSON-RPC doesn't support bigints you will need to transform all the bigints to hex strings here